tailwind.config.js

```diff
@@ -35,12 +35,25 @@ module.exports = {
     },
     screens: {
       sm: "640px",
+      // => @media (min-width: 640px) { ... }
+
+      maxsm: { min: "100px", max: "767px" },
+      // => @media (max-width: 767px) { ... }
+
       md: "768px",
+      // => @media (min-width: 768px) { ... }
+
       lg: "1024px",
+      // => @media (min-width: 1024px) { ... }
+
       xl: "1280px",
+      // => @media (min-width: 1280px) { ... }
+
       "2xl": "1536px",
+      // => @media (min-width: 1536px) { ... }
     },
   },
+
```

Comment:
```suggestion
      md: "768px",
      lg: "1024px",
      xl: "1280px",
      "2xl": "1536px",
    },
  },
```

Reply:
I removed the comments, but I need the custom maxsm class.